src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMonitor.scala

```diff
@@ -130,7 +131,7 @@ class GoogleGroupSyncMonitorActor(
       val groupId: WorkbenchGroupIdentity = parseMessage(message)
 
       groupSynchronizer
-        .synchronizeGroupMembers(groupId)
+        .synchronizeGroupMembers(groupId, samRequestContext = SamRequestContext(None))
```

Comment:
WDYT about starting a trace here? It's an internal kind of request, but I think tracing it and annotating as such could be interesting.

Reply:
New PR/ticket sounds fine. Sorry for ballooning the pr scope :)
We might want it higher up wherever the pub/sub message first gets grabbed from the queue.

Reply:
No worries! I started a trace at the beginning of this case statement. Do we need a trace for the case ReportMessage?